indra/llappearance/llavatarjoint.cpp

```diff
@@ -167,10 +167,10 @@ void LLAvatarJoint::updateJointGeometry()
 }
 
 
-BOOL LLAvatarJoint::updateLOD(F32 pixel_area, BOOL activate)
+bool LLAvatarJoint::updateLOD(F32 pixel_area, bool activate)
 {
-	BOOL lod_changed = FALSE;
-	BOOL found_lod = FALSE;
+	bool lod_changed = FALSE;
```

Comment:
🤔 😄

Reply:
Yeah, that needs to get back in. Thank you for finding it :-D